packages/block-library/src/site-title/edit/index.js

```diff
@@ -31,20 +31,19 @@ export default function SiteTitleEdit( { attributes, setAttributes } ) {
 	} );
 	return (
 		<>
-			<BlockControls>
-				<AlignmentToolbar
-					value={ textAlign }
-					onChange={ ( nextAlign ) => {
-						setAttributes( { textAlign: nextAlign } );
-					} }
-				/>
-
+			<BlockControls group="block">
 				<LevelToolbar
```

Comment:
Inside a "BlockControls" with a "group"  we shouldn't be using `*Toolbar` components otherwise we're adding a extra toolbar group to the Dom, we should be using "*Controls` directly as a `ToolbarGroup` is already rendered by the `BlockControls`

Reply:
Oh... okay. I'll create a follow up. Thanks!

Reply:
Follow up here: https://github.com/WordPress/gutenberg/pull/31032